### DISCUSSION TITLE
Welcome to ArmaPersistentCoreSystem Discussions!

### DISCUSSION THREAD

**Discussion by NikolaiF90** [NikolaiF90/ArmaPersistentCoreSystem#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jschmidt92** [comment]
Hello my name is J. Schmidt (Jacob Schmidt) I've been part of the Arma Community since Arma 2 days. I've done a handful of development projects throughout my time in Arma, and have been working on a major SP/MP framework modification for that last year or so relating to PMC Operations. This framework was influenced by A3M PMC Simulator, CoopR and a few other modifications/missions. Between working on my PMC Framework and a Persistent DB Framework using DragonflyDB and a C# .NET Framework extension I've learned quite a bit and have developed a good list of features.

**Reply by NikolaiF90** [reply]
Hi Jacob. I hope we can work together in the future. If you have any projects you think I can help you wih, please contact me at <NikolaiPrinceF90@gmail.com>
